### DISCUSSION TITLE
Miyoo Mini Plus battery concerns

### DISCUSSION THREAD

**Discussion by CharlieChanba** [OnionUI/Onion#1571] [Q&A] [answered]
Hello everyone,
While it's been advertised that the Miyoo Mini Plus had 7 or more hours of battery life, mine barely goes after 5h.
The battery indicator is very very fluctuent too, it goes from 90% to 85% without playing or turning the console on.
Is there something I'm missing ?I've read that Onion OS has to "learn" to be more precise with the indicator, but I'm on my 4th full charge and nothing seems to be improving. 
If this question has already been ask anywhere, I'm deeply sorry.
Thanks in advance for your answers, and the amazing job that has already been done on Onion OS.

**Comment by Aemiii91** [comment, marked as answer]
5h sounds quite normal. It depends on many factors such as:
1. Wifi
2. Speaker output
3. Which emulator you're running 
4. Screen brightness

I can say we're not utilizing any smart "learning" algorithms in regards to the battery percentage, we simply read the value that's given by the microcontroller.

**Reply by CharlieChanba** [reply]
Thanks a lot for your quick answer ! I'm running mostly Arcade, Snes, TurboGraphx CD and playstation games for the moment, with no wifi and 7/10 brightness. I'm aware this is a bit demanding for this kind of console, but I red on many reddits or sites, people usually get 7-8h of battery running, that's why I prefered to ask here.

Thanks again for your kind answer !

**Reply by Aemiii91** [reply]
No problem. I don't think you can get that much unless you're playing something like GB with headphones.
